junit-runtime/src/main/scala/org/scalajs/junit/JUnitTask.scala

```diff
@@ -3,7 +3,7 @@ package org.scalajs.junit
 import com.novocode.junit.{Ansi, RichLogger}
 import Ansi._
 import sbt.testing._
-import org.scalajs.testinterface.TestUtils
+import org.scalajs.testing.interface.TestUtils
```

Comment:
Ouch, that's actually problematic. It means that a testing framework using `TestUtils` cannot cross-compile for 0.6.x and 1.x anymore. Unless there's an `import` magic that would allow it? But then we should validate it.

Otherwise we need to either introduce type aliases with the new API in 0.6.x, or keep deprecated type aliases for the old API in 1.x.